colcon_cmake/task/cmake/__init__.py

```diff
@@ -206,6 +207,20 @@ def get_project_file(path, target):
     return project_file
 
 
+@functools.lru_cache(maxsize=1)
+def get_msbuild_version():
+    """
+    Get the version of msbuild or throw an exception.
+
+    :rtype: str
+    """
+    cp = subprocess.run(
```

Comment:
Nitpick: please don't abbreviate the variable name since it might not be obvious to everyone what it stands for.